src/Language/PureScript/Docs/Convert/Single.hs

```diff
@@ -66,8 +66,13 @@ type IntermediateDeclaration
 -- since they appear at the top level in the AST, and since they might need to
 -- appear as children in two places (for example, if a data type defined in a
 -- module is an instance of a type class also defined in that module).
+--
+-- The AugmentKindSig constructor allows us to add a kind signature
+-- to its corresponding declaration. Comments for both declarations
+-- are also merged together.
 data DeclarationAugment
   = AugmentChild ChildDeclaration
+  | AugmentKindSig (Maybe Text) P.KindSignatureFor Type'
```

Comment:
Since this is three args, should it be converted into a `KindSigInfo` record type? For example
```haskell
data KindSigInfo = KindSigInfo
  { ksiComms :: Maybe Text
  , ksiKindSig :: P.KindSignatureFor
  , ksiType :: Type'
  }

data DeclarationAugment
  = AugmentChild ChildDeclaration
  | AugmentKindSig KindSigInfo

Reply:
Yes, but can we avoid the abbreviations please? I don't think we need to abbreviate "comments" to "comms", for example.

Reply:
Done.